site/docs/foundations/spacing.mdx

```diff
@@ -79,23 +108,21 @@ Please refer to the [header block pattern page](/salt/patterns/header-block) to
 
 Horizontal spacing between items can vary depending on relationship and context.
 
-Standard spacing between items in a form is typically `--salt-spacing-300`, which aligns with the design systems responsive layout grid, and automatically will adjust depending on what density you use.
-
 | Horizontal spacing | Space between        |
 | ------------------ | -------------------- |
-| Buttons            | `--salt-spacing-100` |
-| Components         | `--salt-spacing-300` |
+| between buttons    | `--salt-spacing-100` |
+| between components | `--salt-spacing-300` |
 
-<Image
-  src="/img/foundations/spacing-components.png"
-  alt="Diagram of what spacing between components should look like."
-/>
+Standard spacing between items in a form pattern is typically `--salt-spacing-300`, which aligns with the design systems responsive layout grid, and automatically will adjust depending on what density you use.
```

Comment:
Can we make 'responsive layout grid' a link to the 'Layout grid' page in Foundations